client/src/components/Email/EmailTemp.js

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import {Container, Dropdown, Form, Grid, Header} from "semantic-ui-react";
+
+const templateOptions = [
+    {
+        key: "SW",
+        text: "Social Worker",
+        value: "1",
+    },
+    {
+        key: "Doc",
+        text: "Doctor",
+        value: "2",
+    }
+];
+
+
+class EmailTemp extends React.Component {
+
+    constructor(props) {
+        super(props);
+        //calling the template options
+
+        this.state = {
+            name: "",
+            lName: "",
+            val: "val",
+            city: "",
+            phone: ""
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleChangeLast = this.handleChangeLast.bind(this);
+        this.handleChangeValue = this.handleChangeValue.bind(this);
+        this.handleChangeCity = this.handleChangeCity.bind(this);
+        // this.handleChangeIllness = this.handleChangeIllness.bind(this);
```

Comment:
remove this comment, it's trivial to add the line back if it's needed in the future